mongodb/install.sls

```diff
@@ -0,0 +1,18 @@
+{% from "mongodb/map.jinja" import mongodb with context %}
+
+add_mongodb_package_repository:
+  pkgrepo.managed:
+    - name: {{ mongodb.repo }}
+    - humanname: MongoDB 2.6.5 Repository
+    - dist: {{ mongodb.repo_dist }}
+    - keyid: {{ mongodb.key }}
+    - keyserver: {{ mongodb.keyserver }}
+    - refresh_db: True
```

Comment:
Also, since there is no repository defined for RedHat this should be placed in a conditional block like
```saltstack
{% if mongodb.repo %}
<do stuff>
{% endif %}
```